manual/manual-fr.adoc

```diff
@@ -0,0 +1,685 @@
+| **Sols Nocifs** |
+
+La lave rouge brûlante et la boue radioactive ne sont que deux des types de sols nocifs que vous pouvez rencontrer dans les niveaux de Freedoom. Si vous devez marcher dessus, essayez de trouver une <<specialitems,combinaison de secours>>, un chemin plus court à travers la zone, ou une façon de traverser la zone sans toucher le sol. |
+image:images/hazard-slime.png[Radioactive slime,150,150,width=100%]
+| **Plafonds Écraseur** |
+
+De nombreux niveaux ont été truqués avec des pièges, et celui-ci n'est qu'un parmi d'autres. Ces plafonds mobiles sont souvent placés au-dessus d'objets tentants. Soyez très prudent de ne pas rester coincé en dessous d'un, sinon il vous écrasera rapidement en bouillie! |
+image:images/hazard-crusher.png[Crushing Ceiling,150,150,width=100%]
+|==========================
+
+=== Mourrir
+
+Éventuellement, vous allez vous retrouver dans une situation que vous ne pouvez pas gérer et votre avatar se fera tuer.
+
+Vous pouvez voir cela comme un signe pour faire une pause, ou recharger votre dernière sauvegarde, ou appuyez sur la touche Utiliser pour redémarrer le niveau avec votre santé complète, mais sans équipements, à part pour votre pistolet et 50 balles. (Certains portages ne font pas ce dernier, à la place ils sauvegardent votre partie au début de chaque niveau, et en appuyant sur la touche Utiliser vous chargez cette partie.)
+
+Il n'y a pas de limite au nombre de vies.
+
+En multijoueur, la touche Utiliser réinitialisera votre santé et votre inventaire et vous placera au départ, mais la partie elle-même continue normalement. Il est possible de ramasser toutes les munitions d'un niveau et de mourir sans les utiliser, forçant votre équipe à terminer le niveau en utilisant uniquement des pistolets, et de mourir afin de recharger.
+
+=== Conseils Tactiques
+
+Si vous gallerez avec la difficulté du jeu, ca vaux peut-ête le coup d'examiner les suggestions suivantes:
+
+* Prenez le temps pour configurer vos contrôles -- pour l'assignation des
+  touches/bouttons et la sensibilité de la souris/manette. Aucune configuration n'est idéale pour tout le monde, c'est donc une bonne idée d'expérimenter: tout ce qui vous aide à esquiver les projectiles tout en gardant votre arme pointée sur l'ennemi, et offre le moins de distraction lorsque vous vous déplacez dans le niveau à la recherche de choses, est positif.
+
+* Jouez avec un casque. La séparation stéréo du jeu peut vous donner des
+  indices audio utiles sur la position des ennemis et vous alerter de projectiles qui se dirigent vers vous. Le casque vous permettra de capter ces signaux de manière plus précise.
+
+* Ne spammez *pas* les boutons! Presque toutes les armes ont un léger délai
+  lorsque vous lâchez la touche Tirer, ce qui prends du temps et donne à votre cible l'opportuniter de riposter. Enfoncer la touche Tirer vous permettra de tirer avec n'importe quelle arme en continu jusqu'à l'épuisement de vos munitions ou que vous lâchiez la touche.
```

Comment:
```suggestion
* Ne spammez *pas* les boutons ! Presque toutes les armes ont un léger délai
  lorsque vous appuyez sur la touche Tirer, ce qui prends du temps et donne à votre cible l'opportuniter de riposter. Enfoncer la touche Tirer vous permettra de tirer avec n'importe quelle arme en continu jusqu'à l'épuisement de vos munitions ou que vous lâchiez la touche.
```

Reply:
The original is wrong then 🙂 That should be changed in its own PR.

Reply:
It's when you let go.

The most obvious are the plasma and BFG - once (only once) you let go of fire you'll see that PLSGB0 sprite which sets a minimum amount of time you have to wait between two short bursts, while the BFG has a similar lag but it doesn't have as obvious a sprite to represent it.